themes/finna2/templates/content/register_details_en-gb.phtml

```diff
@@ -80,11 +79,17 @@
       <li>The source of the viewed record</li>
       <li>Whether the viewed record is available online</li>
   </ul>
+  <h4>Other data</h4>
+  <ul class="list-bullets">
+      <li>Data saved in the cache that is necessary for using the online service</li>
+      <li>The user’s location data used in the search term or location data entered by the user</li>
```

Comment:
En ymmärrä tätä kohtaa. Mitä tarkoittaa "location data used in the search term"?